File: Bank.java
import java.util.ArrayList;

public class Bank {
    private static ArrayList<Employee> employeeList = new ArrayList<>();
    private static ArrayList<BankAccount> accountsList = new ArrayList<>();

    public static ArrayList<Employee> getEmployeeList() {
        return employeeList;
    }

    public static ArrayList<BankAccount> getAccountsList() {
        return accountsList;
    }

    public static void addEmployee(Employee employee){
        employeeList.add(employee);
    }

    public static void addAccountsList(BankAccount account){
        accountsList.add(account);
    }
}

File: Bank.py
class Bank:
    employee_list = []
    accounts_list = []

    @classmethod
    def get_employee_list():
        return Bank.employee_list

    @classmethod
    def get_accounts_list():
        return Bank.accounts_list

    @classmethod
    def add_employee(employee):
        Bank.employee_list.append(employee)

    @classmethod
    def add_accounts_list(account):
        Bank.accounts_list.append(account)